lms/djangoapps/courseware/grades.py

```diff
@@ -842,7 +842,9 @@ def _calculate_score_for_modules(user_id, course, modules):
         if module.category not in ignore_categories and (module.graded or module.has_score):
             module_score = scores_client.get(locations[index])
             if module_score:
-                module_percentages.append(module_score.correct / module_score.total)
+                correct = module_score.correct or 0
```

Comment:
Yes, it is odd, but seems to be normal operation for ScoresClient. If a user hasn't submitted an answer for a module, we get back a "Score" namedtuple with correct and total populated with None.